runners/jet-experimental/src/main/java/org/apache/beam/runners/jet/JetPipelineOptions.java

```diff
@@ -31,6 +31,19 @@
 
   void setJetGroupName(String jetGroupName);
 
+  @Description("Specifies the addresses of the Jet cluster; needed only with external clusters")
+  @Validation.Required
+  String getJetClusterAddresses();
+
+  void setJetClusterAddresses(String jetClusterAddresses);
+
+  @Description(
+      "Specifies where the fat-jar containing all the code is located; needed only with external clusters")
+  @Validation.Required
+  String getCodeJarLocation();
+
+  void setCodeJarLocation(String codeJarLocation);
```

Comment:
It can be an `URL` or a `path string` to a file (on the client machine). I'll try to give it a better name, but I don't see a `resourcesToStage` in `PipelineOptions`. Even the Flink Runner declares his own `filesToStage` in `FilePipelineOptions`. Am I missing something?

Reply:
My bad, I thought this was inherited from the base `PipelineOptions`.